### PR TITLE
Adds gcsweb to istio.io to use its certs

### PR DESCRIPTION
**Pull request by sebastienvas**
Use isito.io nginx proxy to redirect gcsweb.istio.io to in cluster service.

### CODE REVIEW

File: istio.io/configmap-nginx.yaml [hunk 1]
```diff
@@ -149,6 +149,47 @@ data:
...
+          add_header Content-Type text/plain;
+          return 200 "ok\n";
+        }
+
+        location / {
+          proxy_set_header Host gcsweb.istio.io;
+          proxy_set_header X-Real-IP $remote_addr;
+          proxy_set_header X-Forwarded-For $remote_addr;
+          proxy_set_header Connection "";
+          proxy_pass       http://prow;
```
Comment (ayj) [inline istio.io/configmap-nginx.yaml]: s/prow/gcsweb

File: istio.io/configmap-nginx.yaml [hunk 2]
```diff
@@ -149,6 +149,47 @@ data:
...
+      }
+
+      # Handle HTTPS.
+      upstream gcsweb {
+              server    gcsweb.default.svc.cluster.local:80;
+        keepalive 10;
+      }
+
+      server {
+        server_name gcsweb.istio.io;
```
Comment (ayj) [inline istio.io/configmap-nginx.yaml]: We'll need to add gcsweb.istio.io SAN to istio.io cert before this goes live.

File: istio.io/configmap-nginx.yaml [hunk 3]
```diff
@@ -149,6 +149,47 @@ data:
           return 301 https://istio.io$request_uri;
       }
 
+      # gcsweb.istio.io
```
Comment (ayj) [inline istio.io/configmap-nginx.yaml]: Looks like there is a some extra copy/paste errors here. I see two gcsweb.istio.io server blocks.

File: gcsweb.istio.io/service.yaml [hunk 4]
```diff
@@ -7,8 +7,7 @@ metadata:
 spec:
   selector:
     app: gcsweb
-  type: LoadBalancer
-  loadBalancerIP: 104.198.108.251
+  type: NodePort
```
Comment (ayj) [inline gcsweb.istio.io/service.yaml]: Why do we need NodePort here? intra-cluster routing shouldn't need it AFAIK.

File: istio.io/configmap-nginx.yaml (hunk 3 above)
Comment (sebastienvas) [reply istio.io/configmap-nginx.yaml]: I don't see it.

File: istio.io/configmap-nginx.yaml (hunk 2 above)
Comment (sebastienvas) [reply istio.io/configmap-nginx.yaml]: How do we do that ?

**Comment by ldemailly** [comment]
alternative would be to start using cert manager like
https://github.com/istio/admin-sites/tree/master/fortio#fortio-on-istio

which I think we eventually want to move to for all of our sites ?

File: istio.io/configmap-nginx.yaml (hunk 3 above)
Comment (ayj) [reply istio.io/configmap-nginx.yaml]: Nevermind. I forget we defined separate http and https server blocks

File: istio.io/configmap-nginx.yaml (hunk 2 above)
Comment (sebastienvas) [reply istio.io/configmap-nginx.yaml]: Done

**Pull request merged**